pkgs/desktops/gnome/extensions/guillotine/default.nix

```diff
@@ -0,0 +1,53 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitLab,
+  glib
+}:
+
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
```suggestion
# We package this manually because upstream stopped updating the package to
# extensions.gnome.org. See:
# https://gitlab.com/ente76/guillotine/-/issues/17
stdenv.mkDerivation (finalAttrs: {
```